libs/compute_cuda/CMakeLists.txt

```diff
@@ -62,3 +62,9 @@ add_hpx_module(compute_cuda
     hpx_serialization
   CMAKE_SUBDIRS examples tests
 )
+
+if (TARGET hpx::apex)
+  target_link_libraries(hpx_threadmanager PRIVATE hpx::apex)
```

Comment:
Is this what you intended? Shouldn't this be `hpx_compute_cuda` instead?

Reply:
Uh, no... Thanks!